src/crypto/cmac.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * @fileoverview This module implements AES-CMAC on top of
+ * native AES-CBC using either the WebCrypto API or Node.js' crypto API.
+ * @requires asmcrypto.js
+ * @requires util
+ * @module crypto/cmac
+ */
+
+import { AES_CBC } from 'asmcrypto.js/src/aes/cbc/exports';
+import util from '../util';
+
+const webCrypto = util.getWebCryptoAll();
+const nodeCrypto = util.getNodeCrypto();
+const Buffer = util.getNodeBuffer();
+
+
+/**
+ * This implementation of CMAC is based on the description of OMAC in
+ * http://web.cs.ucdavis.edu/~rogaway/papers/eax.pdf. As per that
+ * document:
+ *
+ * We have made a small modification to the OMAC algorithm as it was
+ * originally presented, changing one of its two constants.
+ * Specifically, the constant 4 at line 40 was the constant 1/2 (the
```

Comment:
Line 40 in the paper, right? I realize you are just quoting here, but this is confusing. Can you translate it to the current code?